src/pages/test-report/components/test-report-modal/test-report-modal.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
```

Comment:
Why did we need this extra modal module?

Reply:
To allow the common components of the alert title (icon with title) and the return to test button to be shared between the [ETA modal](https://github.com/dvsa/mes-mobile-app/blob/5d1e635fb5f1150ec136580adfcb578a9c14352e/src/pages/test-report/components/eta-invalid-modal/eta-invalid-modal.module.ts) and the existing [Legal requirements modal](https://github.com/dvsa/mes-mobile-app/blob/5d1e635fb5f1150ec136580adfcb578a9c14352e/src/pages/test-report/components/legal-requirements-modal/legal-requirements-modal.module.ts). The legal requirements modal previously had the title and button code inside the single template. See this [diff](https://github.com/dvsa/mes-mobile-app/pull/488/files/5d1e635fb5f1150ec136580adfcb578a9c14352e?file-filters%5B%5D=.html&file-filters%5B%5D=.scss&file-filters%5B%5D=.ts&owned-by%5B%5D=#diff-af47535884edbddab339db4933a32ebe).